src/cloudai/workloads/deepep/slurm_command_gen_strategy.py

```
@@ -44,96 +98,105 @@ def _append_head_node_detection(self, batch_script_content: List[str]) -> None:
 
-        _, nodes = self.system.get_nodes_by_spec(self.test_run.nnodes, self.test_run.nodes)
-        num_nodes = len(nodes) if nodes else self.test_run.nnodes
+    def _append_cli_field(self, parts: list[str], field_name: str, value: Any) -> None:
+        if value is None or value == "":
+            return
 
-        config_file_path = self.test_run.output_path / "config.yaml"
-        command_parts = [
+        flag = _flag_name(field_name)
+        if isinstance(value, bool):
+            if field_name in _BOOL_VALUE_FIELDS:
```

Ruff (0.15.15)

[error] Undefined name `_BOOL_VALUE_FIELDS`

(F821)